app/code/community/Bolt/Boltpay/Model/Cron.php

```diff
@@ -42,6 +42,8 @@ class Bolt_Boltpay_Model_Cron
      *
      * As an artifact, we leave the parent quotes, (i.e. all Magento created quotes), and converted
      * immutable quotes. We delegate cleanup responsibility of these to the merchants.
+     *
+     * @codeCoverageIgnore Temporarily ignoring to reach 100%. Very difficult to test.
```

Comment:
Let's remove this.  We don't need to get to 100% this way.  We'll do it the difficult way so that our 100% means 100%, and not 100%*

Reply:
Got it.

Reply:
Removed.